### DISCUSSION TITLE
Use LIMIT 5 OFFSET 5

### DISCUSSION THREAD

**Discussion by Dillardz** [pawelsalawa/sqlitestudio#4574] [Q&A]
### Details
Use LIMIT 5 OFFSET 5 however only show 5. It supposed to show 10. Code -> SELECT Name, Continent, Region FROM Country WHERE Continent = 'Europe' ORDER BY Name LIMIT 5 OFFSET 5;

### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.

### Operating system
Windows 10

### SQLiteStudio version
3.4.0

**Comment by pawelsalawa** [comment]
I think you're misusing the `LIMIT` and the `OFFSET`.
The `LIMIT` tells SQLite to output maximum of 5 rows.
The `OFFSET` tells SQLite to skip 5 first rows.

If we assume table `XXX' to be:
| id | letter |
|----|-----|
| 1 | a |
| 2 | b |
| 3 | c |
| 4 | d |
| 5 | e |
| 6 | f |
| 7 | g |
| 8 | h |
| 9 | i |
| 10 | j |
| 11 | k |
| 12 | l |
| 13 | m |
| 14 | n |
| 15 | o |

then  `SELECT * FROM XXX LIMIT 5 OFFSET 5`  will give you:

| id | letter |
|----|-----|
| 6 | f |
| 7 | g |
| 8 | h |
| 9 | i |
| 10 | j |
| 11 | k |
